sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/SchemasImpl.java

```diff
@@ -75,26 +75,34 @@
                 Context context);
 
         @Post("/$schemaGroups/{groupName}/schemas/{schemaName}:get-id")
-        @ExpectedResponses({204, 415})
+        @ExpectedResponses({204})
+        @UnexpectedResponseExceptionType(
+                value = ErrorException.class,
+                code = {415})
         @UnexpectedResponseExceptionType(ErrorException.class)
         Mono<SchemasQueryIdByContentResponse> queryIdByContent(
                 @HostParam("endpoint") String endpoint,
                 @PathParam("groupName") String groupName,
                 @PathParam("schemaName") String schemaName,
                 @QueryParam("api-version") String apiVersion,
-                @BodyParam("application/json; serialization=avro") String schemaContent,
+                @BodyParam("application/json; serialization=Avro") String schemaContent,
```

Comment:
for my own education: if this is eventually ends up in content-type header, why does the case (`avro` -> `Avro`) matter?

Reply:
It doesn't. I can remove it to reduce the diff